versioned/storage/bigtable/src/main/java/org/projectnessie/versioned/storage/bigtable/BigTableBackendFactory.java

```diff
@@ -65,6 +65,13 @@ public static void configureDataClient(BigtableDataSettings.Builder settings) {
       retrySettings.setMaxRetryDelay(maxRetryDelay);
     }
 
+    stubSettings
+        .bulkMutateRowsSettings()
+        .setBatchingSettings(
+            stubSettings.bulkMutateRowsSettings().getBatchingSettings().toBuilder()
+                .setElementCountThreshold((long) BigTableConstants.MAX_BULK_MUTATIONS)
```

Comment:
Do you think it might be worth making this configurable?

Reply:
I thought about that too, but on the other hand, we know our schema, average row sizes etc., so is it really useful to let users tweak these internal things? The retry delay just above is also not configurable currently.

Reply:
Actually, `MAX_BULK_MUTATIONS` is only 1000, so I think we're fine (For some reason I thought it was very large)